crates/oxc_ast/src/ast/js.rs

```diff
@@ -1740,10 +1756,12 @@ pub struct Class<'a> {
     /// @Bar() // <-- Decorator
     /// class Foo {}
     /// ```
+    #[ts]
     pub decorators: Vec<'a, Decorator<'a>>,
```

Comment:
May not be ts, could be stage 3 decorators

Reply:
Acorn [does not seem to support decorators](https://ast.sxzz.moe/#eNolyTEKwzAMQNGrGE0t5ASZCr1C6aRFqCqkOJKxnEIwvnsUsn3+65Bhhh/9ybkupcEEJQaxVY3m6MfX7HZH5Uzu6Zn6wJMsqKOmhCC80luqL6YIc4xMTbwhTJe7bZXltRe5eLXPlqNRB4wDeOQp3w==). So I think we can consider them as TS (for now).